# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/thunderbolt-utils/hotfix-sudo-bash.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./utils.c.vanilla	2024-02-23 10:35:49.400000000 +0000
+++ ./utils.c	2024-02-23 10:36:30.965000000 +0000
@@ -220,7 +220,7 @@
 	char *cmd_to_return = malloc(MAX_LEN * sizeof(char));
 	char cmd_as_root[MAX_LEN];
 
-	snprintf(cmd_as_root, sizeof(cmd_as_root), "sudo bash -c \"%s\"", cmd);
+	snprintf(cmd_as_root, sizeof(cmd_as_root), "sh -c \"%s\"", cmd);
 
 	strncpy(cmd_to_return, cmd_as_root, MAX_LEN);
 	return trim_white_space(cmd_to_return);
